processing/db_handler.py

```diff
@@ -265,12 +275,13 @@ def get_latest_run_with_uuid(self, conn, uuid):
             run_data = cursor.fetchone()
         return run_data
 
-    def get_undone_data_upload_log_o(self,
-                                     conn,
-                                     qaqc_processor_source,
-                                     lookback_h):
+    def get_undone_original_data_upload(self,
```

Comment:
I am still not sure I understand what the function name is describing here.

Reply:
Consider: get_incomplete_original_data_upload